packages/client/src/graphql/users/UserGet.re

```diff
@@ -96,7 +96,7 @@ module Query = [%graphql
 module QueryComponent = ReasonApollo.CreateQuery(Query);
 
 let inner = innerComponentFn => {
-  <QueryComponent>
+  <QueryComponent pollInterval={60 * 1000}>
```

Comment:
Does this need to poll, even if their email has been validated? I wouldn't expect the user model to update much otherwise.

Reply:
No, it does not need to poll if user has been validated. It was the simplest form of updating user info. I sure there are another ways.